### DISCUSSION TITLE
Inspecting a Story with Vue.js devtools

### DISCUSSION THREAD

**Discussion by fprijate** [histoire-dev/histoire#503] [General]
I have noticed that I can inspect Story in devtools only if Story has form:

```
 <Story
     title="Some title"
     :layout="{ type: 'grid', width: ... }"
  > 
```

Regardless of whether the Story has some Variants or not  (only default variant).
If so, there are 3 or more apps in devtools:

1. HistoireApp
2. MountStorySubApp
3. RenderStorySubApp ( one or more : number of variants)

Otherwise, if there is no layout prop or layout type is "single", RenderStorySubApp is/are missing.

**Comment by Akryum** [comment]
We can improve that
